juriscraper/opinions/united_states/federal_special/uscgcca.py

```diff
@@ -0,0 +1,76 @@
+# Scraper for US Coast Guard Court of Criminal Appeals
+
+from juriscraper.lib.string_utils import convert_date_string
+from juriscraper.OpinionSite import OpinionSite
+from re import search
+from requests.utils import requote_uri
+
+
+class Site(OpinionSite):
+    def __init__(self, *args, **kwargs):
+        super(Site, self).__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.mj_cite_regex = "\\d+ *M\\.*J\\.* *\\d+"
```

Comment:
This regex doesn't seem to work.  And if you look at the second page of the website you see citations with and without periods.  I think you want something closer to 
`r"\d{1,4} M\.?J\.? \d{1,4}"`

Reply:
also I think you should denote all regex strings with an r (r"") in python 3.8+